src/test/java/gradle/cucumber/Client.java

```diff
@@ -8,16 +8,24 @@
 
 class Client {
     private final HttpClient client;
+    private HttpRequest.BodyPublisher httpBody;
 
     Client() {
-        client = HttpClient.newHttpClient();
+        this.client = HttpClient.newHttpClient();
+        this.httpBody = HttpRequest.BodyPublishers.noBody();
     }
 
     HttpResponse<String> request(String port, String method, String location) throws IOException, InterruptedException {
         HttpRequest request = HttpRequest.newBuilder()
             .uri(URI.create("http://localhost:" + port + location))
-            .method(method, HttpRequest.BodyPublishers.noBody())
+            .version(HttpClient.Version.HTTP_1_1)
```

Comment:
I think setting the version makes sense to keep in the builder as it rarely changes for specific requests